theme/templates/resource-landing-page/top-right-buttons.html

```diff
@@ -87,6 +87,17 @@
                         {% endif %}
                     {% endif %}
                 {% endif %}
+
+                <button id="btn-download-all" class="tr-download-button"
+                    title="Download all content as Zipped BagIt Archive"
+                    {% if cm.raccess.require_download_agreement %}
+                    data-target="#license-agree-dialog-bag" data-toggle="modal"
+                    data-placement="auto"
+                    {% endif %}
+                    data-bag-url="{{ cm.bag_url }}">
+                    <span class="fa fa-cloud-download icon-blue"></span>
+                </button>
```

Comment:
this should probably best be a `<a>` instead of `<button>`
see the other buttons next to it and mimic them. If you do that, you won't need to create a custom `.tr-download-button` class and you can get rid of the extra css